src/glib/mine/signalproc.h

```diff
@@ -773,6 +773,10 @@ class TAggrResampler {
     TStr RoundStart;
     /// Default value when the buffer is empty
     TFlt DefaultVal;
+    /// Which side of bin interval is closed. The default is ‘left’.
```

Comment:
Bucket intervals are half open. If Closed=`left`, the interval includes the left time point and is open on the right side (and the oposite if Closed=`right`).